WordPress/Classes/Models/Post.swift

```diff
@@ -195,7 +195,7 @@ class Post: AbstractPost {
 
         if let originalPost = original as? Post {
 
-            if tags?.characters.count != originalPost.tags?.characters.count && tags != originalPost.tags {
+            if tags ?? "" != originalPost.tags ?? "" {
```

Comment:
I see your point, the only problem is that with https://github.com/wordpress-mobile/WordPress-iOS/pull/6521 I've already added two more fields to be compared, and if you look at AbstractPost there's like 10 more there. I would like to avoid having 10+ functions to compare 10+ fields and a big || for all of them. Furthermore the functions would only be used here. I can't come up with anything smarter/more elegant though.